composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "derhansen/fe_change_pwd",
+  "name": "derhansen/fe-change-pwd",
```

Comment:
The package name is intended to be `derhansen/fe_change_pwd` and this will not be changed.